cms/djangoapps/contentstore/signals/handlers.py

```diff
@@ -159,11 +160,42 @@ def listen_for_course_publish(sender, course_key, **kwargs):  # pylint: disable=
     transaction.on_commit(lambda: emit_catalog_info_changed_signal(course_key))
 
 
+def _determine_producer_config_for_signal_and_topic(signal, topic):
```

Comment:
Can this be deduplicated into some shared module? Maybe temporarily have an `openedx/core/lib/events.py` or something.

Reply:
I was being lazy because it's temporary but i can DRY it out

Reply:
[fixed]